packages/babel-plugin-proposal-decorators/src/index.ts

```diff
@@ -7,6 +7,7 @@ import {
   FEATURES,
 } from "@babel/helper-create-class-features-plugin";
 import legacyVisitor from "./transformer-legacy";
+import transformer2020_09 from "./transformer-2020-09";
```

Comment:
Nit: We can rename this to `2021-12`, since it's when it was last presented (without `@init`).